packages/modals/src/styled/StyledClose.ts

```diff
@@ -5,22 +5,42 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
+import styled, { css, DefaultTheme, ThemeProps } from 'styled-components';
+import rgba from 'polished/lib/color/rgba';
 import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'modals.close';
 
-export interface IStyledCloseProps {
-  hovered?: boolean;
-}
+const colorStyles = (props: ThemeProps<DefaultTheme>) => {
+  const baseColor = getColor('primaryHue' as string, 600, props.theme);
+
+  return css`
+    background-color: transparent;
+    color: ${getColor('neutralHue' as string, 600, props.theme)};
+
+    &:hover {
+      background-color: ${rgba(baseColor as string, 0.08)};
+      color: ${getColor('neutralHue' as string, 700, props.theme)};
+    }
+
+    &[data-garden-focus-visible] {
+      box-shadow: ${props.theme.shadows.md(rgba(baseColor as string, 0.35))};
```

Comment:
Do not use `rgba` directly. Instead, prefer to call `getColor('primaryHue', 600, props.theme, 0.35)`

Reply:
I originally called `getColor` instead of `rgba`, but opted to follow how `StyledButton.ts` [created it's `box-shadow`](https://github.com/zendeskgarden/react-components/blob/next/packages/buttons/src/styled/StyledButton.ts#L71). What is the heuristic that can be used to determine whether to use `rgba` versus `getColor`?

I've updated this branch to use `getColor`, and removed `polished/rgba` 👍.  The demo has also been updated.

Reply:
The opacity enhancement was added to `getColor` after buttons was converted and I missed updating that spot. Use `getColor`.